tests/test_ble_lifecycle_controller.py

```
@@ -0,0 +1,671 @@
+
+        mock_ownership = SimpleNamespace(
+            _discard_invalidated_connected_client=_discard_invalidated_connected_client
+        )
+
+        controller = BLELifecycleController(mock_iface)
+        controller._connection_ownership = mock_ownership  # type: ignore[assignment]
+
+        with (
+            patch(
+                "meshtastic.interfaces.ble.lifecycle_service.BLELifecycleService._state_manager_transition_to",
+                side_effect=lambda iface, state: state == ConnectionState.DISCONNECTED,
```

Ruff (0.15.6)

[warning] Unused lambda argument: `iface`

(ARG005)